test.ts

```diff
@@ -0,0 +1,48 @@
+import { Selector } from 'testcafe';
+
+fixture `SFX Web Components Functional Testing`
+      .page `./demo/demo.html`;
+
+const searchbox = Selector('sfx-search-box');
+const clearButton = Selector('.sfx-clear');
+const autocomplete = Selector('sfx-autocomplete');
+
+test('SFX search populates sayt terms and sayt products', async t => {
+  await t
+    .setTestSpeed(0.4)
+    .hover(searchbox)
+    .click(searchbox)
+    .pressKey( 'home left c h e c k space d r e s s' )
+});
+
+test('SFX sayt items are hoverable', async t => {
+await t
+    .setTestSpeed(0.4)
+    .hover(searchbox)
+    .click(searchbox)
+    .pressKey( 'home left c h e c k space d r e s s' )
+    .setTestSpeed(0.6)
```

Comment:
Can you speak to the need to set the test speed, and to do so potentially more than once during a test?

Reply:
Sure, I found that certain actions were happening too quickly to register (tp the eye) so I slowed some down to different degrees. I can change of course if you like